src/helpers/geetest.js.txt

```diff
@@ -0,0 +1,281 @@
+// /* eslint-disable */
```

Comment:
why do we need this to be merged right now to master? If it's some code for the future, make proper PR with it

Reply:
I don't think we should be afraid hiding that code written in very bad style. It will be replace to newer one